compiledb/parser.py

```diff
@@ -35,6 +35,9 @@
 make_enter_dir = re.compile(r"^\s*make\[\d+\]: Entering directory [`\'\"](?P<dir>.*)[`\'\"]\s*$")
 make_leave_dir = re.compile(r"^\s*make\[\d+\]: Leaving directory .*$")
 
+# We want to skip such lines from configure to avoid spurious MAKE expansion errors.
+checking_make = re.compile(r"^checking whether .* sets \$\(MAKE\)... (yes|no)$")
```

Comment:
Could you generalize it a bit more?
Something like this would be ok:
```python
checking_make = re.compile(r"^checking whether .* sets \$\(\w+\)\.\.\. (yes|no)$")
```

Reply:
Sounds sensible, adapted PR uploaded...